code/modules/vehicles/vehicle_actions.dm

```diff
@@ -364,6 +364,41 @@
 	passtable_off(rider, VEHICLE_TRAIT)
 	vehicle.pass_flags &= ~PASSTABLE
 
+/datum/action/vehicle/ridden/scooter/skateboard/kflip
+	name = "Kickflip"
+	desc = "Kick your board up and catch it."
+	button_icon_state = "skateboard_ollie"  // youre still going up
+	check_flags = AB_CHECK_CONSCIOUS
+
+/datum/action/vehicle/ridden/scooter/skateboard/kflip/Trigger()
+	var/obj/vehicle/ridden/scooter/skateboard/V = vehicle_target
+	var/mob/living/L = owner
+
+	L.adjustStaminaLoss(V.instability)
+	if (L.getStaminaLoss() >= 100)
+		playsound(src, 'sound/effects/bang.ogg', 20, TRUE)
+		V.unbuckle_mob(L)
+		L.Paralyze(50)
+		if(prob(15))
+			V.visible_message(span_userdanger("You smack against the board, hard."), \
+			span_danger("[L] misses the landing and falls on [L.p_their()] face!)"))
+			L.emote("scream")
+			L.adjustBruteLoss(10)  // thats gonna leave a mark
+			return
+		V.visible_message(span_userdanger("You fall flat onto the board!"), \
+		span_danger("[L] misses the landing and falls on [L.p_their()] face!"))
+	else
```

Comment:
trailing else case. Instead, make the above if block always return, and unindent the entire else block + remove else